docs/field-fft.ipynb

```diff
@@ -0,0 +1,668 @@
+    "\n",
+    "However, we can `translate` the mesh to move the field back to the desired position `inplace`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "f03d1d98",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<strong>Mesh</strong>\n",
+       "<ul>\n",
+       "  <li><strong>Region</strong>\n",
+       "<ul>\n",
+       "  <li>pmin = [0.0, 0.0, -3.308722450212111e-24]</li>\n",
+       "  <li>pmax = [5e-09, 2e-08, 4.5000000000000006e-08]</li>\n",
+       "  <li>dims = ['x', 'y', 'z']</li>\n",
+       "  <li>units = ['m', 'm', 'm']</li>\n",
+       "</ul></li>\n",
+       "  <li>n = [5, 20, 45]</li>\n",
+       "  </ul>"
+      ],
+      "text/plain": [
+       "Mesh(Region(pmin=[0.0, 0.0, -3.308722450212111e-24], pmax=[5e-09, 2e-08, 4.5000000000000006e-08], dims=['x', 'y', 'z'], units=['m', 'm', 'm']), n=[5, 20, 45])"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "ifft_field.mesh.translate(field.mesh.region.centre, inplace=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93900090",
+   "metadata": {},
+   "source": [
+    "As a check we can see if our original field is the same as the field which has been Fourier transformed and then inverse Fourier transformed. By using the `allclose` method in the `Field` class we can see that the two field are almost the same."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "f791993e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "field.allclose(ifft_field)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9340b200",
+   "metadata": {},
+   "source": [
+    "While our field has been transformed back into its original space, using the `==` operator reveals that the initial and final states are not identical."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "de0a5492",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "False"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "field == ifft_field"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2f867782",
+   "metadata": {},
+   "source": [
+    "This discrepancy stems primarily from two factors:\n",
+    "\n",
+    "1. **Floating-Point Precision**: Arithmetic operations on floating-point numbers can lead to small imprecisions.\n",
+    "\n",
+    "2. **Complex Values**: Initially, our field comprised real values. However, the processing via FFT and inverse FFT results in a field with complex values. \n",
```

Comment:
Maybe we can add two additional plots to show that the real part is mostly the initial field and the imaginary part practically zero?